### ISSUE TITLE
Error uploading Iris dataset

### ISSUE THREAD

**Issue by DraXus** [EpistasisLab/pennai#208]
I get the following error when I try to upload [this iris.csv file](https://gist.githubusercontent.com/curran/a08a1080b88344b0c8a7/raw/d546eaee765268bf2f487608c537c05e22e4b221/iris.csv) and set "species" as dependent:

`Unable to upload file. Error: Datafile validation failed, sklearn.check_array() validation could not convert string to float: 'virginica'`

![image](https://user-images.githubusercontent.com/2436/58158070-f7b17500-7c79-11e9-98d2-65bc6859e065.png)

I tried also to set "species" as categorical feature but got this error: `Unable to upload file. Error: Datafile validation failed, Target column 'species' cannot be a categorical feature`.

I'm using [Pennai 0.13](https://github.com/EpistasisLab/pennai/releases/tag/v0.13). Is this a bug or am I doing something wrong?

**Comment by lacava** [comment]
hi @DraXus, thanks for catching this! At the moment we aren't processing non-numeric class labels. But we will push an update very soon that handles this case.

**Comment by DraXus** [comment]
Oh, good to know! Thanks for your quick answer @lacava, I'll close this issue then 👍

**Comment by weixuanfu** [comment]
The 'target_column' should be dropped before [this line](https://github.com/EpistasisLab/pennai/blob/v0.13/lab/pyutils/validateDataset.py#L101). Maybe adding the line below will solve the issue:

```
num_df.drop(columns=target_column, inplace=True)
```

**Comment by hjwilli** [comment]
Thanks @weixuanfu, that was part of the solution!

An update has been pushed that allows the target column to contain strings.
